admin-dev/themes/new-theme/js/app/pages/translations/store/actions.js

```diff
@@ -106,16 +106,17 @@ export const saveTranslations = ({ commit }, payload) => {
       successfullySaved: translations.length,
       store: payload.store,
     });
-    payload.store.state.modifiedTranslations = [];
+    commit(types.RESET_MODIFIED_TRANSLATIONS);
     return showGrowl('notice', 'Translations successfully updated');
   }, (error) => {
     showGrowl('error', error.bodyText ? JSON.parse(error.bodyText).error : error.statusText);
   });
 };
 
-export const resetTranslation = ({ commit }, payload) => {
-  const url = payload.url;
-  const translations = payload.translations;
+/* eslint-disable-next-line no-unused-vars */
+export const resetTranslation = ({commit}, payload) => {
+  const {url} = payload;
```

Comment:
```suggestion
  const {url, translations} = payload;
```
Isn't it possible to do that?

Reply:
yes it is, but it's due to the autofix :)

Reply:
Autofix doesnt understand properties repeated... badly :'(

Reply:
This is something we can improve later